src/endpoints/discovery.ts

```diff
@@ -0,0 +1,25 @@
+import { ConnectorImagesSpecResponse } from 'endpoints/connectors';
+import { client } from 'services/client';
+import { BaseResponse } from 'types';
+
+export interface DiscoveryResponse extends BaseResponse {
+    data: {
+        id: string;
+        type: string;
+        attributes: {
+            bindings: any[];
+        };
+        links: {
+            image: string;
+        };
+    };
+}
+
+export const discoveryEndpoint = {
+    create: (
+        endpoint: ConnectorImagesSpecResponse['data']['links']['discovery'],
+        data: any
+    ) => {
+        return client<DiscoveryResponse>(endpoint, { data });
+    },
+};
```

Comment:
This is an example where the CRUD naming kind of falls down. It feels weird to say we are "creating" a discovery... but we kind of are... but it still feels weird.

Reply:
Still need to make a request type... will handle that later.